src/components/TooltipNew/PopperTooltip.js

```diff
@@ -274,6 +296,14 @@ class PopperTooltip extends React.PureComponent {
           [className]: className
         })}
       >
+        {closeWithX && (
+          <div
```

Comment:
one last thing: can we use `button` instead of div? I would like to focus on this close button with `tab` key

Reply:
I changed it to button

Reply:
And I provided padding `4px`